pokedex/data/csv/pokemon_species.csv

```diff
@@ -897,3 +897,10 @@ id,identifier,generation_id,evolves_from_species_id,evolution_chain_id,color_id,
 896,glastrier,8,,474,9,8,,-1,3,35,0,120,0,1,0,1,0,896,
 897,spectrier,8,,475,1,8,,-1,3,35,0,120,0,1,0,1,0,897,
 898,calyrex,8,,476,5,12,,-1,3,100,0,120,0,1,0,1,0,898,
+899,wyrdeer,8,234,,4,,,4,,,0,,,1,,0,0,899,
```

Comment:
Hi, here the data lacks information. By building the DB I get that:

> null value in column "forms_switchable" of relation "pokemon_v2_pokemonspecies" violates not-null constraint

I guess we should try to fill all the columns.